app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
```

Comment:
- Kindly define a new comment method as you did with the post controller. It must be something like this:
   def new
    @comment = Comment.new
  end

Reply:
### Implementation considerations
- because in the project requirement did not specify a new view for the post, comments, and like. I implemented like any other blog Page 
  - the posting form in a new view
  - the comment form in the show post page (same controller)
  - like is just a button